docs/configuration/integrations/email.md

```diff
@@ -0,0 +1,28 @@
+# Emails
+
+## SMTP Configuration
+
+Follow the instructions below to set up the Codacy integration with SMTP to send emails:
+
+1.  Edit the file `values-production.yaml`, set `global.email.enabled: "true"` and define the remaining values with the credentials for your smtp server:
+
+    ```yaml
+    email:
+        enabled: "true"
+        replyTo: "notifications@mycompany.com"
+        smtp:
+            hostname: "smtp.mycompany.com"
+            protocol: "smtps" # smtps | smtp
+            username: "my-smtp-username"
+            password: "a-s!r0ng-pVsSwORG"
```

Comment:
Can we have comments to guide the user setting this up in the same style as we have for other configurations in the `values-production.yaml`, for example https://github.com/codacy/chart/blob/master/codacy/values-production.yaml#L114 ?